pom.xml

```diff
@@ -45,6 +44,22 @@
 
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-remote</artifactId>
+        <version>${version.org.kie}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>openshift-kie-hacep</artifactId>
+        <version>${version.org.kie}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>sample-hacep-project-kjar</artifactId>
+        <version>${version.org.kie}</version>
+      </dependency>
+
       <dependency>
         <groupId>io.thorntail</groupId>
         <artifactId>bom-all</artifactId>
```

Comment:
I know this is not part of this PR but I would like not to import thorntail bom in top level project. Can you please move it to thorntail module? I expect it is needed only there